samples/msal-angular-v2-samples/angular10-browser-sample/src/app/home/home.component.ts

```diff
@@ -7,9 +8,10 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HomeComponent implements OnInit {
 
-  constructor() { }
+  constructor(private authService: MsalService) { }
 
   ngOnInit(): void {
+    this.authService.handleRedirectObservable();
```

Comment:
I think it would be good to call subscribe and console.log the result/error at least.

Reply:
Done